content/projects/gsoc/2019/remoting-over-apache-kafka-docker-k8s-features.adoc

```diff
@@ -15,15 +15,67 @@ mentors:
 - "oleg_nenashev"
 links:
   draft: https://docs.google.com/document/d/1CIgjz8IZxihYC6CKqE0pFuKljCKK3SPhKK87IpHs_38
+  github: "jenkinsci/remoting-kafka-plugin"
 ---
 
-Current version of Remoting over Apache Kafka plugin requires users to manually configure the entire system which includes Zookeeper, Apache Kafka and Remoting agents. It also doesn't support dynamic agent provisioning so scalability is harder to achieve. This project aims to solve two problems:
+I am Long Nguyen from https://international.fpt.edu.vn/[FPT University], Vietnam. My work during Google Summer of Code 2019 is to integrating plugin:remoting-kafka[Remoting over Apache Kafka plugin] with Kubernetes environment and later release link:/blog/2019/08/19/remoting-kafka-kubernetes-release-2/[version 2.0].
```

Comment:
It is a project page, not a blogpost. No need in self-intro, your author page is linked anyway